service-app/internal/api/handler.go

```diff
@@ -308,6 +308,9 @@ func (c *IndexController) IngestHandler(w http.ResponseWriter, r *http.Request)
 		c.logger.InfoWithContext(reqCtx, "No errors found!", nil)
 	}
 
+	// Clear errors
+	c.Queue.ClearErrors()
```

Comment:
I was going to suggest we need it on line 306 too, to avoid it combining errors, but I guess that's true of earlier returns too.Maybe we should add it at the start of the function under a `defer`?